code/datums/diseases/advance/symptoms/clockwork.dm

```diff
@@ -0,0 +1,357 @@
+/datum/symptom/robotic_adaptation
+	name = "Biometallic Replication"
+	desc = "The virus can manipulate metal and silicate compounds, becoming able to infect robotic beings. The virus also provides a suitable substrate for nanites in otherwise inhospitable hosts"
+	stealth = 1
+	resistance = -2
+	stage_speed = 4 //while the reference material has low speed, this virus will take a good while to completely convert someone
+	transmittable = 0
+	level = 9
+	severity = 0
+	symptom_delay_min = 10
+	symptom_delay_max = 60
+	var/replaceorgans = FALSE
+	var/replacebody = FALSE
+	var/robustbits = FALSE
+	threshold_desc = "<b>Stage Speed 4:</b>The virus will replace the host's organic organs with mundane, biometallic versions. +1 severity.<br>\
+                      <b>Stage Speed 10:</b>The virus will eventually convert the host's entire body to biometallic materials, and maintain its cellular integrity. +1 severity.<br>\
+                      <b>Stage Speed 14:</b>Biometallic mass created by the virus will be superior to typical organic mass. -3 severity."
+
+/datum/symptom/robotic_adaptation/OnAdd(datum/disease/advance/A)
+	A.infectable_biotypes |= MOB_ROBOTIC
+
+/datum/symptom/robotic_adaptation/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.properties["stage_rate"] >= 5) //at base level, robotic organs are purely a liability
```

Comment:
```suggestion
	if(A.properties["stage_rate"] >= 4) //at base level, robotic organs are purely a liability
```

Reply:
It's funny, this was the value i originally had, but i wanted to change it up. While purely even, semi-regular intervals are nice, i like these scores more